lib/src/layer/tile_layer/tile_coordinates.dart

```diff
@@ -3,25 +3,32 @@ import 'dart:math';
 import 'package:meta/meta.dart';
 
 @immutable
```

Comment:
This class is no longer immutable.

```suggestion
```

Reply:
removed caching. The new hash is super cheap anyways